app/common/messages.ts

```diff
@@ -0,0 +1,182 @@
+/**
+ * This module contains all common types for backend and frontend
+ * IPC communication
+ */
+
+/** Version of the protocol */
+export const IPC_VERSION = 1
```

Comment:
After talking a bit we took the decision (@aesedepece and myself) of removing the version field from the protocol because at this stage it doesn't seem to be that useful.